extensions/amp-consent/0.1/amp-consent.js

```diff
@@ -637,6 +643,63 @@ export class AmpConsent extends AMP.BaseElement {
     return this.remoteConfigPromise_;
   }
 
+  /**
+   * TODO (micajuineho): Use our stored info to check if we have the
+   * necessary granular consents.
+   * @param {ConsentInfoDef} opt_consentInfo
+   * @return {!Promise<boolean>}
+   */
+  checkGranularConsentRequired_(opt_consentInfo) {
```

Comment:
1. `opt_consentInfo` looks unused. Assuming it will be used later?
2. More of an AMP practice question - instead of using `opt_` prefix, can we use default params?

Reply:
Yup! Unused for now, but we're going to use it in the future to determine if the stored purpose consents match with the publisher defined `purposeConsentRequired` field. If all the purposes have consents, then we can unblock, otherwise we should should the consent prompt. 

> More of an AMP practice question - instead of using opt_ prefix, can we use default params?

Both are allowed in the codebase. There's also the `unused-` prefix.